src/Components/DonationFormModal.js

```diff
@@ -1,7 +1,46 @@
 import React, { useState } from 'react';
+import { gql, useMutation  } from '@apollo/client';
 import '../Styles/DonationFormModal.scss';
 
-const DonationFormModal = () => {
+const CREATE_PET = gql`
```

Comment:
Mutation to create a pet that's available for adoption. Much smoother/quicker since it was our second mutation.